cmd/internal/printconfig/command.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package printconfig
+
+import (
+	"fmt"
+	"sort"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func printConfigurations(cmd *cobra.Command, v *viper.Viper, includeEmpty bool) {
+	keys := v.AllKeys()
+	sort.Strings(keys)
+
+	for _, key := range keys {
+		value := v.Get(key)
+		source := "default"
+		if v.IsSet(key) {
+			source = "user-assigned"
+		}
+
+		if includeEmpty {
+			fmt.Fprintf(cmd.OutOrStdout(), "%s: %v=%s\n", source, key, value)
```

Comment:
it may be more readable it you use padding to print this as a table. I would place default/user-assigned as the last column

Reply:
Good suggestion, I've implemented a more organised output and updated the PR again